Basics/Add a string PY/index.js

```diff
@@ -0,0 +1,14 @@
+function Add() {
+  let input = document.getElementById("string").value;
```

Comment:
Since input to constant, it's value doesn't changes later, we better use const here.
We use const when the value doesn't change later and use let when value is changed later.